### ISSUE TITLE
scene.assets is None when load a scene

### ISSUE THREAD

**Issue by HAWLYQ** [google-research/kubric#256]
Hi, thanks for your wonderful work! When exploring this tool, I found that **scene.assets is empty when loading an existing scene**. I'm sure there are multiple objects in the scene because there is no problem rendering pictures. However,  **scene.assets is empty** can cause that the data_stack['segmentation'] is a zero matrix.** 
You can reproduce this problem as follows:

    scene = kb.Scene(resolution=(512, 512), frame_start=$start_frame$, frame_end=$end_frame$) 
    blend_path = $blend_path$" # blend path of an existing scene
    renderer = KubricRenderer(scene, custom_scene=blend_path) 
    print(len(scene.assets))

I will appreciate very much for your suggestions.

**Comment by Qwlouse** [comment]
Yes that is indeed a problem. The way kubric is designed it currently only knows about objects that it has added itself. So things in the scene, or added directly via `bpy` are invisible to kubric. 
I agree though that this is an important usecase that we should try to support. 

My suggestion would be to add a wrapper object that one could create manually around existing blender objects to make them visible to Kubric. I think in its basic form this should be rather straightforward to do. Let me know If you would be interested in trying your hand at an implementation. Otherwise I'll add it to my backlog of feature requests (which is unfortunately growing rather long).

**Comment by MrXandbadas** [comment]
I've been looking into it as I agree it would be an important part of the pipeline. There has been a few times where I could have used the object details from pre-made blender files instead of ensuring a complete build through worker files.

@Qwlouse I will be looking into its implementation. What are your suggestions/thoughts on it?
